routes/admin.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+var Boom = require('boom');
+var knex = require('../connection');
+var getParents = require('../services/admin').getParents;
+var formatBox = require('../services/admin').formatBOX;
+Promise = require('bluebird');
+
+module.exports = [
+  {
+    /**
+     * @api {get} /admin/:level/units List Admins
+     * @apiGroup Admin
+     * @apiName ListAdmins
+     * @apiDescription Returns list of admin unit ids of specified level.
+     * @apiVersion 0.1.0
+     *
+     * @apiParam {String} level Admin level (province, district, or commune)
+     *
+     * @apiSuccess {JSON} units object including list of all admin ids in the level supplied in parameters
+     * @apiSuccess {string} units.level admin units' level
+     * @apiSuccess {array} units.units list of objects representing all units in level. each object icludes an id and name
+     *
+     *
+     * @apiSuccessExample {JSON} Example Usage:
+     *  curl http://localhost:4000/api/admins/commune
+     *
+     * @apiSuccessExample {JSON} Success-Response
+     * {
+     *  level: 'province',
+     *  'units': [
+     *    {
+     *      id: 106,
+     *      name_en: 'Bac Ninh
+     *    }
+     *    ...
+     *  ]
+     * }
+    */
+    method: 'GET',
+    path: '/admin/{level}/units',
+    handler: function (req, res) {
+      const level = req.params.level;
+      // check to make sure admin level is of the proper type
+      const levelCheck = ['commune', 'district', 'province'].find((l) => { return l === level; });
+      if (!levelCheck) {
+        res(Boom.badRequest('Admin level must be of type \'province\', \'district\', or \'commune'));
```

Comment:
Have you checked whether the rest of the code below will still run if this conditional is triggered?  I'm not sure if you have to `return res(Boom.badRequest)` or not.